test-poc/providers/pom.xml

```diff
@@ -0,0 +1,71 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+    <parent>
+        <artifactId>keycloak-test-parent</artifactId>
+        <groupId>org.keycloak.test</groupId>
+        <version>999.0.0-SNAPSHOT</version>
+        <relativePath>../pom.xml</relativePath>
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>providers</artifactId>
+    <name>Keycloak Test Framework Custom Providers</name>
+    <packaging>jar</packaging>
+    <description>Deploying and testing custom providers with the new test framework</description>
+    <version>${parent.version}</version>
+
+    <dependencies>
+        <dependency>
+            <groupId>org.keycloak</groupId>
+            <artifactId>keycloak-test-framework</artifactId>
+            <version>999.0.0-SNAPSHOT</version>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
```

Comment:
@vaceksimon  these three can be left once you move the pom module under showcase, other dependencies can be removed and reused from parent